### DISCUSSION TITLE
Welcome to Merliot!

### DISCUSSION THREAD

**Discussion by scottfeldman** [merliot/hub#8] [Announcements]
### Welcome to Merliot!

**Comment by joeblew999** [comment]
Hey Scott 

I wanted to work on the web aspects to use htmx and sse , so it’s real time collab.

could not see the code .

the web GUI colours are very loud - reminds me of Star Trek

**Reply by scottfeldman** [reply]
Hi Joe, the project already uses htmx and htmx-websockets.  See github.com/merliot/hub: pkg/device/**.

**Comment by joeblew999** [comment]
Thanks will try it out .

I love the colours btw … 

Using this for htmx : https://github.com/starfederation/datastar

it uses SSE rather than WS …
